Library/Homebrew/cmd/bottle.rb

```diff
@@ -102,6 +102,8 @@ def print_filename(string, filename)
         if link.to_s.start_with?(string)
           absolute_symlinks_start_with_string << pn
         else
+          # allow absolute symlinks to system frameworks
+          next if link.to_s.start_with?("/System/")
```

Comment:
It feels like this would look more natural if placed before `if link.to_s.start_with?(string)` or is the concern that this could somehow clash with `string` and lead to incorrect results?